contracts/src/management/ManagementContract.sol

```diff
@@ -23,10 +23,13 @@ contract ManagementContract is Initializable, OwnableUpgradeable {
     // Event to log changes to important contract addresses
     event ImportantContractAddressUpdated(string key, address newAddress);
 
-    mapping(address => string) private attestationRequests;
+    // mapping of enclaveID to attestation request report
+    //mapping(address => string) private attestationRequests; todo: do we need to store this mapping?
```

Comment:
We have been storing the full attestation report in the contract state, but it's not read at the moment. The enclave gets the data from the transaction calldata currently.

Are there reasons it's a good idea to store these attestation reports on the L1?

Reply:
I think it's a vestige from the time we were planning to perform the verification on-chain.
Can be removed